src/components/reusable/CityItem/CityItem.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Grid from '@material-ui/core/Grid';
+import WeatherIcon from '../../reusable/WeatherIcon/WeatherIcon';
+import { Link } from "react-router-dom";
+import {
```

Comment:
how about just **import styles from './CityItem.scss'** ? ...and place it at the end of import block